src/bioregistry/data/curated_mappings.sssom.tsv

```diff
@@ -56,60 +97,117 @@ bioregistry:gold	Not	skos:exactMatch	lov:gold	orcid:0000-0003-4423-4370	semapv:M
 bioregistry:gro	Not	skos:exactMatch	biocontext:GRO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:gro	Not	skos:exactMatch	miriam:gro	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:gro	Not	skos:exactMatch	obofoundry:gro	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
+bioregistry:gtdb	Not	skos:exactMatch	pathguide:190	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:hdl		skos:exactMatch	fairsharing:FAIRsharing.0b7e54	orcid:0009-0009-5240-7463	semapv:MappingReview	
 bioregistry:hp	Not	skos:exactMatch	aberowl:HP_O	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration	
 bioregistry:hp	Not	skos:exactMatch	bioportal:HP_O	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration	
 bioregistry:hp	Not	skos:exactMatch	lov:hpo	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration	
 bioregistry:hpath	Not	skos:exactMatch	fairsharing:FAIRsharing.kj336a	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
+bioregistry:icdc	Not	skos:exactMatch	fairsharing:FAIRsharing.d95034	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:icdc	Not	skos:exactMatch	re3data:r3d100010405	orcid:0009-0009-5240-7463	semapv:MappingReview	
 bioregistry:icdo	Not	skos:exactMatch	aberowl:ICDO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:icdo	Not	skos:exactMatch	bioportal:ICDO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:icdo	Not	skos:exactMatch	ontobee:ICDO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
+bioregistry:igsn	Not	skos:exactMatch	n2t:igsn	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:inn		skos:exactMatch	wikidata:P2275	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:intact	Not	skos:exactMatch	togoid:Intact	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:interpro		skos:exactMatch	wikidata:P2926	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:ipi		skos:exactMatch	integbio:nbdc00911	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:iuphar.receptor		skos:exactMatch	uniprot:DB-0182	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:iuphar.receptor		skos:exactMatch	wikidata:P5458	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:jax	Not	skos:exactMatch	rrid:IMSR_JAX	orcid:0009-0009-5240-7463	semapv:MappingReview	
 bioregistry:kegg.reaction	Not	skos:exactMatch	wikidata:P665	orcid:0000-0001-9439-5346	semapv:MappingReview	
+bioregistry:knapsack		skos:exactMatch	wikidata:P2064	orcid:0009-0009-5240-7463	semapv:MappingReview	
+bioregistry:lcsh	Not	skos:exactMatch	wikidata:P244	orcid:0009-0009-5240-7463	semapv:MappingReview	
```

Comment:
I think this mapping is basically correct, although the Library of Congress's "naming authority" IDs are somewhat broader than "subject headings" and can include people, organizations, etc. In fact the Bioregistry record for `lcsh` curated by @cthoyt uses (nb2018006591)[http://id.loc.gov/authorities/names/nb2018006591] as an example LUI which corresponds to ISB (International Society for Biocuration) and is technically not a "subject heading". So perhaps in this case, changes to the Bioregistry entry to broaden the scope might make sense. 

```suggestion
bioregistry:lcsh	skos:exactMatch	wikidata:P244	orcid:0009-0009-5240-7463	semapv:MappingReview	
```